dts/bindings/pinctrl/nuvoton,numaker-pinctrl.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Nuvoton Technology Corp. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Pin controller is responsible for controlling pin function
+  selection and pin properties. For example, for example you can
+  use this node to set UART0 RX as pin PB12 to fulfill SYS_GPB_MFP3_PB12MFP_UART0_RXD.
+
+  The node has the 'pinctrl' node label set in your SoC's devicetree,
+  so you can modify it like this:
+
+    &pinctrl {
+            /* your modifications go here */
+    };
+
+  All device pin configurations should be placed in child nodes of the
+  'pinctrl' node, as shown in this example:
+
+    &pinctrl {
+      /* configuration for the uart0 "default" state */
+      uart0_default: uart0_default {
+          /* configure PB13 as UART0 TX and PB12 as UART0 RX */
+          pinmux = <PB12MFP_UART0_RXD 0x0000>, <PB13MFP_UART0_TXD 0x0000>;
+      };
+    };
+
+
+  To link pin configurations with a device, use a pinctrl-N property for some
+  number N, like this example you could place in your board's DTS file:
+
+    #include "board-pinctrl.dtsi"
+
+    &uart0 {
+        pinctrl-0 = <&uart0_default>;
+        pinctrl-names = "default";
+    };
+
+compatible: "nuvoton,numaker-pinctrl"
+
+include:
+  - name: base.yaml
+  - name: pincfg-node.yaml
+    child-binding:
+      property-allowlist:
+        - drive-push-pull
+        - drive-open-drain
+        - bias-disable
+        - bias-pull-down
+        - bias-pull-up
+
+child-binding:
+  description: |
+    Each child node defines the configuration for a particular state.
+  properties:
+    pinmux:
+      required: true
+      type: array
+      description: |
+        An array of pins sharing the same group properties. The pins should
+        be defined using pre-defined macros or, alternatively, using NVT_PINMUX
+        macros depending on the pinmux model used by the SoC series.
+    drive-strength:
+      type: string
+      default: "low"
+      enum:
+        - "low"
+        - "fast"
+      description: |
+        Set the driving strength of a pin. Default to low is enough to drive most
+        components, like as LED, CAN transceiver and son on.
+    slew-rate:
+      type: string
+      default: "fast"
+      enum:
+        - "fast"
+        - "low"
+      description: |
+        Set the speed of a pin. This setting effectively limits the
+        slew rate of the output signal. Default to fast for the capability to
+        support high speed pins, like as SPI CLK up to 50MHz.
```

Comment:
What is hardware default? `fast` default is, in general, not desirable as it can cause EMI issues if the circuit is not designed for high speed in mind.

Reply:
Done